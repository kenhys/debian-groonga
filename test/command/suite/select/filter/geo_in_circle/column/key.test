table_create area TABLE_PAT_KEY WGS84GeoPoint
column_create area index COLUMN_INDEX area _key

load --table area
[
{"_key": "128592911x503145263"},
{"_key": "128565076x502976128"},
{"_key": "128461363x502772717"},
{"_key": "128597458x502942345"},
{"_key": "128573438x502868189"}
]

select area --filter 'geo_in_circle(_key, "128575000x502960000", 3000)'
