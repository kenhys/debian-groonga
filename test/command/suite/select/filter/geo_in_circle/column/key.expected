table_create area TABLE_PAT_KEY WGS84GeoPoint
[[0,0.0,0.0],true]
column_create area index COLUMN_INDEX area _key
[[0,0.0,0.0],true]
load --table area
[
{"_key": "128592911x503145263"},
{"_key": "128565076x502976128"},
{"_key": "128461363x502772717"},
{"_key": "128597458x502942345"},
{"_key": "128573438x502868189"}
]
[[0,0.0,0.0],5]
select area --filter 'geo_in_circle(_key, "128575000x502960000", 3000)'
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        3
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "WGS84GeoPoint"
        ],
        [
          "index",
          "area"
        ]
      ],
      [
        5,
        "128573438x502868189",
        1
      ],
      [
        2,
        "128565076x502976128",
        1
      ],
      [
        4,
        "128597458x502942345",
        1
      ]
    ]
  ]
]
